jobs/JGDAS_ATMOS_GLDAS

```diff
@@ -0,0 +1,136 @@
+# Source machine runtime environment
+##########################################
+# shellcheck disable=SC1090-SC1091
+. "${HOMEgfs}/env/${machine:?}.env" gldas
+# shellcheck disable=
+status=$?
+[[ ${status} -ne 0 ]] && exit "${status}"
+
+
+##############################################
+# Obtain unique process id (pid) and make temp directory
+##############################################
+export pid=${pid:-$$}
+export outid=${outid:-"LL${job}"}
+
+export DATA=${DATA:-${DATAROOT}/${jobid:?}}
+mkdir -p "${DATA}"
+cd "${DATA}" || exit
+
+
+##############################################
+# Run setpdy and initialize PDY variables
+##############################################
+export cycle="t${cyc}z"
+setpdy.sh
+# shellcheck disable=SC1091
+. ./PDY
+# shellcheck disable=
+
+
+##############################################
+# Determine Job Output Name on System
+##############################################
+export pgmout="OUTPUT.${pid}"
+export pgmerr=errfile
+
+
+##############################################
+# Set variables used in the exglobal script
+##############################################
+export CDATE=${CDATE:-${PDY}${cyc}}
+export CDUMP=${CDUMP:-${RUN:-"gdas"}}
+export COMPONENT=${COMPONENT:-atmos}
+
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+export gldas_ver=${gldas_ver:-v2.3.0}
+export HOMEgldas=${HOMEgldas:-${HOMEgfs}}
+export FIXgldas=${FIXgldas:-${HOMEgldas}/fix/fix_gldas}
+export PARMgldas=${PARMgldas:-${HOMEgldas}/parm/gldas}
+export EXECgldas=${EXECgldas:-${HOMEgldas}/exec}
+export USHgldas=${USHgldas:-${HOMEgldas}/ush}
+export PARA_CONFIG=${HOMEgfs}/parm/config/config.gldas
+
+if [[ "${RUN_ENVIR}" = "nco" ]]; then
+    export COMIN=${COMIN:-${ROTDIR}/${RUN}.${PDY}/${cyc}/${COMPONENT}}
+    export COMOUT=${COMOUT:-${ROTDIR}/${RUN}.${PDY}/${cyc}/${COMPONENT}}
+else
+    export COMIN="${ROTDIR}/${CDUMP}.${PDY}/${cyc}/${COMPONENT}"
+    export COMOUT="${ROTDIR}/${CDUMP}.${PDY}/${cyc}/${COMPONENT}"
+fi
+if [[ ! -d ${COMOUT} ]]; then
+    mkdir -p "${COMOUT}"
+    chmod 775 "${COMOUT}"
+fi
+
+export COMINgdas=${COMINgdas:-${ROTDIR}}
+export DCOMIN=${DCOMIN:-${DCOMROOT:-"/lfs/h1/ops/prod/dcom"}}
+
+export model=${model:-noah}
+export MODEL=${MODEL:-"${model} |tr 'a-z' 'A-Z'"}
+
+
+###############################################################
+# Run relevant exglobal script
+
+${GLDASSH:-${HOMEgldas}/scripts/exgdas_atmos_gldas.sh}
+status=$?
+[[ ${status} -ne 0 ]] && exit "${status}"
+
+##############################################
+# End JOB SPECIFIC work
+##############################################
+
+##############################################
+# Final processing
+##############################################
+if [[ -e "${pgmout}" ]] ; then
+  cat "${pgmout}"
+fi
+
+##########################################
+# Remove the Temporary working directory
+##########################################
+cd "${DATAROOT}" || exit
```

Comment:
Theoretically so should this one, but I don't know if we want to job to "fail" it it just can't delete `${DATA}`.

Reply:
Added "1" with the exit:

```
cd "${DATA}" || exit 1
```